.github/workflows/cherry-pick-pr.yml

```diff
@@ -12,62 +12,9 @@ permissions:
   pull-requests: write
 
 jobs:
-  load-config:
```

Comment:
I have removed the references from the config file, and I have used the references in the file directly.

The goal was to simplify the workflows as much as possible.